api/nakadi-event-bus-api.yaml

```diff
@@ -613,6 +632,15 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: partition
```

Comment:
could you elaborate on that, please? it is not clear why we want partition in params if we have it in path already.